### ISSUE TITLE
module.exports documented as global members

### ISSUE THREAD

**Issue by csnover** [jsdoc/jsdoc#667]
Given:

``` js
/**
 * @module test
 */

/**
 * Description.
 *
 * @enum {string}
 */
module.exports = {
  'a': 'A',
  'b': 'B'
};
```

Expected:
Module "test" is documented with description "Description" and the enum properties table.

Actual:
Module "test" is documented with no description and static members "a" and "b", and global "module:test" is documented with the description and the enum properties table.

This seems to be an issue with any `module.exports = (object literal)`, the exports object is showing up in globals. Indirection with `var exports = (object literal); module.exports = exports;` partially fixes the problem in the above test case (module then gets the enum table plus all the static members) but causes some infinite loop inside jsdoc in my actual project.

Version: 3.3.0-alpha8 in Node.js

**Comment by srad** [comment]
Is this really fixed? Im using JSDoc 3.3.0-beta3 and still get the members shown as "global" memebers. Example:

```javascript
define(function () {
    /**
     * @exports some/type
     * @enum {string}
      */
    var exports = {
        A: 'abc'
    };

    return exports;
});
```

**Comment by hegemonic** [comment]
@srad That looks like a different bug. Please file a new issue.
